### ISSUE TITLE
Have a benchmark for AutoML

### ISSUE THREAD

**Issue by ddutta** [mlcommons/training_policies#28]
Have a simple starter benchmark for comparing different AutoML algorithms e.g. from http://www.ml4aad.org/literature-on-neural-architecture-search/ 

Supporting orgs: Cisco

**Comment by petermattson** [comment]
Hi Debo, this would be separate from MLPerf training benchmark suite correct? Just supported by MLPerf org?

**Comment by ddutta** [comment]
Yes ... it would cover things like: ENAS, MnasNet, AmoebaNet, DARTS etc.

**Comment by ShriyaRishab** [comment]
Clean up, closing very old issues.
